x/market/keeper/keeper.go

```diff
@@ -52,14 +52,16 @@ func (k Keeper) CreateOrder(ctx sdk.Context, gid dtypes.GroupID, spec dtypes.Gro
 		OrderID: types.MakeOrderID(gid, oseq),
 		Spec:    spec,
 		State:   types.OrderOpen,
-		StartAt: ctx.BlockHeight() + orderTTL, // TODO: check overflow
+		StartAt: ctx.BlockHeight() + orderTTL,                         // TODO: check overflow
+		CloseAt: ctx.BlockHeight() + orderTTL + spec.OrderBidDuration, // TODO: check overflow, set via parameter
```

Comment:
This is a critical issue now because `spec.OrderBidDuration` is supplied by the user instead of being a constant (`5`).